build.sbt

```diff
@@ -51,13 +52,13 @@ lazy val draft_api = (project in file(".")).
       "org.apache.logging.log4j" % "log4j-slf4j-impl" % Log4JVersion,
       "org.scalikejdbc" %% "scalikejdbc" % "2.5.0",
       "org.postgresql" % "postgresql" % "9.4-1201-jdbc4",
-      "mysql" % "mysql-connector-java" % "5.1.36",
       "com.amazonaws" % "aws-java-sdk-s3" % AwsSdkversion,
       "org.scalaj" %% "scalaj-http" % "2.3.0",
       "org.elasticsearch" % "elasticsearch" % ElasticsearchVersion,
       "com.sksamuel.elastic4s" %% "elastic4s-core" % Elastic4sVersion,
       "com.sksamuel.elastic4s" %% "elastic4s-http" % Elastic4sVersion,
       "com.sksamuel.elastic4s" %% "elastic4s-aws" % Elastic4sVersion,
+      "com.fasterxml.jackson.core" % "jackson-databind" % JacksonVersion, // Overriding jackson-databind used in elastic4s because of https://snyk.io/vuln/SNYK-JAVA-COMFASTERXMLJACKSONCORE-32111
```

Comment:
I linken så står det "Upgrade com.fasterxml.jackson.core:jackson-databind to version 2.8.11.1, 2.9.5 or higher."
Noen grunn til at du har gått for `2.9.4`?

Reply:
2.9.5 var ikke sluppet når jeg sjekket, men ser den har kommet nå, så det kan nok endres ja...

Reply:
@jnatten Bedre nå? :)

Reply:
:+1: